cmd/beach/cmd/helpers.go

```diff
@@ -177,6 +177,12 @@ func startLocalBeach() error {
 				return errors.New("failed to check for database server container health")
 			}
 			if strings.TrimSpace(output) == "healthy" {
+				commandArgs := []string{"exec", "local_beach_database", "mariadb-upgrade", "-u", "root", "--password=password"}
```

Comment:
This runs the upgrade – on every start. But since it does no harm to run it when not needed, I chose this way.

An alternaive would be to check if the upgrade is needed (using `mariadb-upgrade --check-if-upgrade-is-needed`), but that seemed superfluous.